ROADMAP.md

```diff
@@ -37,7 +55,6 @@ Reference: [ModelMesh issues](https://github.com/kserve/modelmesh-serving/issues
 
 ## Objective: "Graduate InferenceGraph to beta"
```

Comment:
@rachitchauhan43 Anything to update for InferenceGraph?

Reply:
Hey @yuzisun, nothing that I could think of apart from making RawDeployment mode more robust.

@bmopuri : Anything on top of your mind ?

Reply:
I know @israel-hdez has been looking into the IG support. So tagging him for feedback here.